aria_extension_tests/cloudify_aria_adapters/test_context_adapter.py

```diff
@@ -304,6 +303,13 @@ def mock_workflow(ctx, graph):
                     operation_name,
                     operation_kwargs=op_dict
                 )
+                if inputs:
```

Comment:
Note that this is a new change that Ran added in his CLI work to validate workflow inputs.